test/release/examples/primers/associative.compopts

```diff
@@ -0,0 +1 @@
+-sparSafeDefault
```

Comment:
If possible, please modify the `associative` primer to:
* work with -snoParSafeWarning instead of -sparSafeDefault, and
* talk about the lack of par-safety

Rationale: we want release/examples generally to use only stable features.